src/services/events/events-router.ts

```diff
@@ -83,4 +101,22 @@ eventsRouter.delete("/:EVENTID", async (req, res, next) => {
     }
 });
 
+eventsRouter.post("/check-in", async (req, res, next) => {
+    try {
+        const { eventId, userId } = req.body;
+        const result = await checkInUserToEvent(eventId, userId);
+        if (result.success) {
+            return res
+                .status(StatusCodes.OK)
+                .json({ message: "Check-in successful" });
+        } else {
+            return res
+                .status(StatusCodes.NOT_FOUND)
+                .json({ error: result.message });
+        }
```

Comment:
clean this up a little, based on changes to cIUTE

Reply:
okay this is weird to do because I have to check if result is an array. Is that okay, or can i keep as is?